data/npc/scripts/bank.lua

```diff
@@ -0,0 +1,318 @@
+				npcHandler:say("Here you are.", cid)
+			else
+				npcHandler:say("Sorry, you do not have enough platinum coins.", cid)
+			end
+		else
+			npcHandler:say("Well, can I help you with something else?", cid)
+		end
+		npcHandler.topic[cid] = 0
+	elseif msgcontains(msg, "change crystal") then
+		npcHandler:say("How many crystal coins would you like to change into platinum?", cid)
+		npcHandler.topic[cid] = 21
+	elseif npcHandler.topic[cid] == 21 then
+		if getMoneyCount(msg) < 1 then
+			npcHandler:say("Sorry, you do not have enough crystal coins.", cid)
+			npcHandler.topic[cid] = 0
+		else
+			count[cid] = getMoneyCount(msg)
+			npcHandler:say("So you would like me to change " .. count[cid] .. " of your crystal coins into " .. count[cid] * 100 .. " platinum coins for you?", cid)
+			npcHandler.topic[cid] = 22
+		end
+	elseif npcHandler.topic[cid] == 22 then
+		if msgcontains(msg, "yes") then
+			if player:removeItem(2160, count[cid])  then
+				player:addItem(2152, count[cid] * 100)
+				npcHandler:say("Here you are.", cid)
+			else
+				npcHandler:say("Sorry, you do not have enough crystal coins.", cid)
+			end
+		else
+			npcHandler:say("Well, can I help you with something else?", cid)
+		end
+		npcHandler.topic[cid] = 0
+	end
+	return true
+end
+
+keywordHandler:addKeyword({"money"}, StdModule.say, {npcHandler = npcHandler, text = "We can {change} money for you. You can also access your {bank account}."})
+keywordHandler:addKeyword({"change"}, StdModule.say, {npcHandler = npcHandler, text = "There are three different coin types: 100 gold coins equal 1 platinum coin, 100 platinum coins equal 1 crystal coin. So if you'd like to change 100 gold into 1 platinum, simply say '{change gold}' and then '1 platinum'."})
+keywordHandler:addKeyword({"advanced"}, StdModule.say, {npcHandler = npcHandler, text = "Your bank account will be used automatically when you want to {rent} a house or place an offer on an item on the {market}. Let me know if you want to know about how either one works."})
+keywordHandler:addKeyword({"help"}, StdModule.say, {npcHandler = npcHandler, text = "You can check the {balance} of your bank account, {deposit} money or {withdraw} it. You can also {transfer} money to other characters, provided that they have a vocation."})
+keywordHandler:addKeyword({"functions"}, StdModule.say, {npcHandler = npcHandler, text = "You can check the {balance} of your bank account, {deposit} money or {withdraw} it. You can also {transfer} money to other characters, provided that they have a vocation."})
+keywordHandler:addKeyword({"basic"}, StdModule.say, {npcHandler = npcHandler, text = "You can check the {balance} of your bank account, {deposit} money or {withdraw} it. You can also {transfer} money to other characters, provided that they have a vocation."})
+keywordHandler:addKeyword({"job"}, StdModule.say, {npcHandler = npcHandler, text = "I work in this bank. I can change money for you and help you with your bank account."})
```

Comment:
multi-line, please!

Reply:
can you give me one sample please? Everything I tried looked so awful!

Reply:
I'd try:
```lua
keywordHandler:addKeyword({"job"}, StdModule.say, {
  npcHandler = npcHandler,
  text = "I work in this bank. I can change money for you and help you with your bank account.",
})
```